test/ci-scripts/Write-TestResults.ps1

```diff
@@ -468,15 +468,6 @@ switch ($CredScanResult) {
     }
 }
 
-switch ($TemplateAnalyzerResult) {
```

Comment:
I think we should keep this code - that way when we add the badges to the readme it's simply a doc change and the readme change - we won't need to change the pipeline or scripts (except for the docs below).  Rather than piecemeal the badges...